src/components/Posts/PostItem.tsx

```diff
@@ -156,7 +160,16 @@ const PostItem = ({ post }: PostProps) => {
           >
             {post.numberOfLikes}
           </Button>
-          <IconButton>
+          <IconButton
+            onClick={() => {
+              copyToClipboard();
+              dispatch(
```

Comment:
I noticed you moved the dispatch calls in `PostMenu` and `ExpandedPostItem` inside of `copyToClipboard()` and I think we should do the same here just for consistency